src/ZipkinOpenTracing/NoopSpanContext.php

```diff
@@ -1,47 +0,0 @@
-<?php
-
-namespace ZipkinOpenTracing;
-
-use ArrayIterator;
-use Zipkin\Propagation\DefaultSamplingFlags;
-use OpenTracing\SpanContext as OTSpanContext;
-
-class NoopSpanContext implements OTSpanContext, WrappedTraceContext
-{
-    public static function create()
-    {
-        return new self();
-    }
-
-    /**
-     * @inheritdoc
-     */
-    public function getContext()
```

Comment:
The reason why I created this class was because of this method. Since `noop` and `real` context should have the same public interface and this method is important (because it returns the surrogate context), not having this class could break stuff (if it does not, probably my bad for not covering this with tests). This is also the reason for `NoopSpan` to exists in this library.